core/build.gradle

```diff
@@ -37,6 +40,42 @@ def getTimestamp() {
     return df.format(today)
 }
 
+ext {
+    repoPath = 'web3j/web3j-docs'
```

Comment:
Maybe change the variable name to `docsPath` or something similar showing that this is for docs.